src/js/input-range/input-range.jsx

```diff
@@ -51,6 +59,7 @@ export default class InputRange extends React.Component {
       disabled: false,
       maxValue: 10,
       minValue: 0,
+      minSize: 1,
```

Comment:
That's a nice idea / suggestion. But I think we might have a problem with `minSize` set to `1` by default. This is because `step` value might not be `1`. We want `minSize` to equal to `step` or exact multiples of `step`. Do you know how to resolve that problem?